src/main/java/softdesign/image/ImageArrayListStore.java

```diff
@@ -1,33 +0,0 @@
-package main.java.softdesign.image;
```

Comment:
Can this be put back into the `image` package?

Reply:
same as above